apps/framework-cli/src/cli/routines/streaming.rs

```diff
@@ -1,36 +1,48 @@
-use log::debug;
-use pathdiff::diff_paths;
-use std::collections::HashMap;
-use std::path::PathBuf;
-use std::{fs, io::Write, process::Stdio};
-
 use crate::cli::display::{Message, MessageType};
 use crate::framework::core::code_loader::{
     load_framework_objects, FrameworkObject, FrameworkObjectVersions,
 };
 use crate::framework::core::infrastructure::table::ColumnType;
+use crate::framework::languages::SupportedLanguages;
+use crate::framework::python::templates::PYTHON_BASE_STREAMING_FUNCTION_TEMPLATE;
 use crate::framework::typescript::templates::TS_BASE_STREAMING_FUNCTION_TEMPLATE;
 use crate::project::Project;
+use log::debug;
+use pathdiff::diff_paths;
+use std::collections::HashMap;
+use std::path::PathBuf;
+use std::{fs, io::Write, process::Stdio};
 
 use super::{RoutineFailure, RoutineSuccess};
 
 pub struct StreamingFunctionFileBuilder {
+    language: SupportedLanguages,
     function_file_path: PathBuf,
     function_file_template: String,
 }
 
 impl StreamingFunctionFileBuilder {
```

Comment:
Could you use this as an opportunity to remove the Object pattern for this routine?